packages/css-data/src/parse-css.ts

```diff
@@ -31,22 +50,14 @@ const unprefixProperty = (property: string) => {
   return property.replace(prefixRegex, "");
 };
 
-const camelCaseProperty = (property: string) => {
-  let camelcased = camelCase(property);
-  if (property[0] === "-") {
-    camelcased = camelcased[0].toLocaleUpperCase() + camelcased.slice(1);
```

Comment:
this used to turn
-webkit-something into WebkitSomething

Reply:
Now it doesn't

Reply:
well unless there was a bug, at least this was the intention

Reply:
Well, prefixed properties parsed incorrectly with it 🤷